packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -46,6 +47,11 @@ export const OdieSendMessageButton = () => {
 	const inputRef = useRef< HTMLTextAreaElement >( null );
 	const attachmentButtonRef = useRef< HTMLElement >( null );
 	const { trackEvent, chat, addMessage, isUserEligibleForPaidSupport } = useOdieAssistantContext();
```

Comment:
```suggestion
	const { trackEvent, chat, addMessage, isUserEligibleForPaidSupport, canConnectToZendesk } = useOdieAssistantContext();
```
We can use the value from the context here if you prefer

Reply:
Yes thank you! I like it more using it from the context